src/components/Flyout/Flyout.stories.tsx

```diff
@@ -33,10 +33,10 @@ const FlyoutTemplate: Story<FlyoutProps> = (args) => {
 
     return (
         <div className="dark:tw-text-white">
-            <div className="tw-flex tw-items-cente">
+            <div className="tw-flex tw-items-center">
                 Some text
                 <Flyout {...args}>
-                    <div className="tw-flex tw-flex-col tw-gap-y-8">
+                    <div className="tw-flex tw-flex-col tw-gap-y-8  tw-p-8">
```

Comment:
```suggestion
                    <div className="tw-flex tw-flex-col tw-gap-y-8 tw-p-8">
```
Could you run prettier on the file just to be sure everything is formatted?